src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1140,7 +1140,7 @@
     },
     {
       "run": "after",
-      "snippetPath": "itemStatisticalCodeReferenceCheckTrigger.sql",
+      "snippetPath": "StatisticalCodeReferenceCheckTrigger.sql",
       "fromModuleVersion": "22.1.0"
```

Comment:
We shouldn't really change existing upgrade scripts (I wish we used tooling like liquibase that better described this). It is misleading and can lead to confusion when folks try to interpret these scripts in the context of versions.

I imagine that these changes aren't really intended to be applied to version 22.1.0 and later (given that 23.0.0 was the last release) which is the impression this change gives.

Rather we should add a new upgrade script that makes the changes from this point onwards.